src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,100 @@
 package com.theironyard.novauc;
 
+import spark.ModelAndView;
+import spark.Session;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
 public class Main {
+    static HashMap<String, User> users = new HashMap<>();
+    static ArrayList<Form> messages = new ArrayList<>();
 
     public static void main(String[] args) {
-	// write your code here
+        Spark.init();
+
+
+
+        Spark.get("/", ((request, response) -> {
+                    HashMap m = new HashMap();
+                    Session session = request.session();
+                    String name = session.attribute("userName");
+                    User user = users.get(name);
+
```

Comment:
Messages DO NOT SHOW UP!  Fix that and resubmit.  Also please make sure edit and delete work before you resubmit.